app/views/users/show.html.erb

```diff
@@ -1,7 +1,39 @@
 <h1> <%= @user.username.capitalize %>'s Profile </h1>
 <ul>
   <li>Country of origin: <%= @user.country_of_origin %></li>
-  <li>Gender: <%= @user.gender  %></li>
+  <% unless @user.gender == "Not specified" %>
+    <li>Gender: <%= @user.gender  %></li>
+  <% end %>
   <li>About: <%= @user.about %></li>
-
 </ul>
+
+<h2>Reviews</h2>
+<% @user.reviews.each do |review| %>
+<div>
+  <h4>Review for: <%= review.city.name %>, <%= review.city.country.name %></h4>
+  <li>
+    <%= "Food: " + "#{review.food_rating}" %>
```

Comment:
it would be more idiomatic if expressed this way:

```erb
Food <%= review.food_rating %>
```

For field names and value pairs, consider a `<dl>` or definition list html element.